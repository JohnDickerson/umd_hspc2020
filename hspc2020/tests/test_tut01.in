7

3 1 1 1

3 1 1 +

5 5 2 R D +

8 5 -2 4 R D 9 + +

7 2 2 + + + D 2

11 2 R 2 R 2 R 2 R 2 R 2

5 -2 2 + D 0
