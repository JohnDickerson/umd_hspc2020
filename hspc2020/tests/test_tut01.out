3
4
30
27
46
2
0
